From ea42953d3d75278f4f89009dfeed115200e4d102 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Wed, 10 Jun 2020 17:13:43 +0000
Subject: [PATCH 2/2] Fix manpage

Use the right stylesheet for DocBook 4. This fixes the bad whitespace
processing.

Use the just-built xsltproc to generate the manpage.
---
 doc/Makefile.am  | 4 ++--
 doc/xsltproc.xml | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 3d5364ca..6a7b9408 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -239,9 +239,9 @@ libxslt-api.xml libxslt-refs.xml EXSLT/libexslt-api.xml \
 	($(XSLTPROC) -o ../libxslt/libxslt.syms $(srcdir)/syms.xsl $(srcdir)/symbols.xml)
 
 xsltproc.1: $(srcdir)/xsltproc.xml
-	-@(if [ -x $(XSLTPROC) ] ; then \
+	-@(if [ -x $(srcdir)/../xsltproc/xsltproc ] ; then \
 	   echo "Rebuilding the man pages from the xml" ; \
-	   $(XSLTPROC) --nonet $(srcdir)/xsltproc.xml ; fi)
+	   $(srcdir)/../xsltproc/xsltproc --nonet $(srcdir)/xsltproc.xml ; fi)
 
 # Note that in the following, xmllint output is piped to xsltproc
 search.php: $(srcdir)/api.xsl $(srcdir)/site.xsl $(srcdir)/search.templ \
diff --git a/doc/xsltproc.xml b/doc/xsltproc.xml
index 08fc3e97..6ca1b323 100644
--- a/doc/xsltproc.xml
+++ b/doc/xsltproc.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <?xml-stylesheet type="text/xsl"
-   href="http://cdn.docbook.org/release/xsl/current//manpages/docbook.xsl"?>
+   href="http://cdn.docbook.org/release/xsl-nons/current/manpages/docbook.xsl"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN"
     "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" [
     
-- 
2.27.0

